prime-router/src/main/kotlin/tokens/Scope.kt

```diff
@@ -16,26 +15,71 @@ class Scope {
 
         /**
          * Scope that represents prime admin access.
+         * Pattern is <organization>.<receiver-or-sender>.<DetailedScope>
+         * The first value must always be an organization name, or wildcard
+         * The idea is to leave the last field loose; it could be a role or a permission
+         * Examples:
+         * md-phd.*.user -- user level access
+         * md-phd.default.report -- able to submit reports only
+         * md-phd.*.admin
+         * *.*.primeadmin
          */
-        val primeAdminScope = "primeadmin.all.all"
+        val primeAdminScope = "*.*.primeadmin"
```

Comment:
Could be moved to an authConstants file with other auth constant values, so all hard coded constants for auth are in one location.

Reply:
I'll leave it here because this is a scope and this is the Scope class.